plugins/mcreator-localization/help/fr_FR/potion/bad.md

```diff
@@ -1,3 +1,3 @@
-Check this parameter in case if your effect is bad for the player. 
+Cochez ce paramètre au cas où votre effet serait mauvais pour le joueur.

```

Comment:
You did not listen to me...
Correction: "Cochez ce paramètre si votre effet est mauvais pour le joueur."